static/scripts/dropzone/dropzone.js

```diff
@@ -932,6 +932,7 @@
       var dirReader, errorHandler, readEntries;
       dirReader = directory.createReader();
       errorHandler = function(error) {
+        // TODO: verified if we need this console log, why not use sentry 
```

Comment:
again, only client-side. I don't see a problem with this, since Sentry also monitors the console

Reply:
Sentry already monitored the browser console.

Reply:
true